mirascope/chat/models.py

```diff
@@ -20,7 +30,7 @@ def __init__(self, model: str = "gpt-3.5-turbo", api_key: Optional[str] = None):
 
     def create(
         self,
-        prompt: Prompt,
+        prompt: Union[Prompt, str],
```

Comment:
What if instead of prompt, they want to pass in messages directly through kwargs. so `prompt: Union[Prompt, str, None]` and then if prompt is None check if messages is inside kwargs

Reply:
Good call. Done.